action.yml

```diff
@@ -83,7 +83,16 @@ runs:
         ${INPUT_MODULE:+-m $INPUT_MODULE} \
         ${{ inputs.args }}; } | tee "${tmpout}"
 
-        (echo -n "stdout=" && cat "${tmpout}") >> "$GITHUB_OUTPUT"
+        {
+          # we need a delim that doesn't appear in the output - a hash of the
+          # file itself *probably* won't (if it does, we have larger
+          # cryptographic problems)
+          delim="$(sha256sum $tmpout | cut -d " " -f1)"
+          echo "stdout<<${delim}"
+          cat "${tmpout}"
+          echo
+          echo "${delim}"
```

Comment:
That is very creative

Reply:
:stuck_out_tongue_winking_eye: hehe ye, better suggestions appreciated, I didn't want to use a fixed `EOF` string on the off chance that that appeared in the output.